kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/requirement/KubernetesRequirement.java

```diff
@@ -0,0 +1,37 @@
+package org.arquillian.cube.kubernetes.impl.requirement;
+
+import com.squareup.okhttp.OkHttpClient;
+import com.squareup.okhttp.Request;
+import com.squareup.okhttp.Response;
+import io.fabric8.kubernetes.client.DefaultKubernetesClient;
+import io.fabric8.kubernetes.client.HttpClientAware;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import org.arquillian.cube.spi.requirement.Requirement;
+import org.arquillian.cube.spi.requirement.UnsatisfiedRequirementException;
+
+import java.io.IOException;
+
+//TODO: The kubernetes client currently doesn't expose a method to do a version check. An issue has been raised, but until its done we do the work here. See https://github.com/fabric8io/kubernetes-client/issues/477.
```

Comment:
@lordofthejars let's create an issue here out of it and reference to the linked one. Otherwise it might get lost.